src/components/NavigationBar.js

```diff
@@ -1,24 +1,103 @@
 import React from 'react';
-
-import './NavigationBar.css';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
 
 import wbooksLogo from '../assets/wbooks_logo.svg';
 import notificationsSvg from '../assets/notifications.svg';
 import addBookSvg from '../assets/add_book.svg';
 
-function NavigationBar() {
-  return (
-    <div className="navigation-bar">
-      <div className="nav-bar-logo-container">
-        <img src={wbooksLogo} alt="WBooks" className="navigation-logo" />
-      </div>
-      <div className="nav-bar-buttons-container">
-        <button src={notificationsSvg} alt="Notifications" className="nav-button nav-notifications-button" />
-        <button src={addBookSvg} alt="Add Book" className="nav-button nav-add-book-button" />
-        <button src={notificationsSvg} alt="Settings" className="nav-button nav-avatar-button" />
+import Dropdown from './Dropdown';
+import './NavigationBar.css';
+
+const axiosInstance = axios.create({
+  baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1/'
+});
+
+const logout = (onSuccess, onError) => {
```

Comment:
move all the interactions with the autentication api to a service. we usually create services in `src/services` and divide them using models logic. i.e: AuthService, BooksService, CommentsService, etc.